framework/source/class/qx/util/StringEscape.js

```diff
@@ -37,26 +37,21 @@ qx.Bootstrap.define("qx.util.StringEscape",
     {
       var entity, result = "";
 
-      for (var i=0, l=str.length; i<l; i++)
-      {
-        var chr = str.charAt(i);
-        var code = chr.charCodeAt(0);
+      for (var codePoint of str) {
```

Comment:
Unfortunately, you need to use the old syntax for the time being, because we said we would keep supporting the Python generator for v6, and it doesn't understand ECMAScript 2015 features such as `for ... of`